sdk/OoyalaSkinSDK/EndScreen.js

```diff
@@ -65,8 +65,6 @@ var EndScreen = React.createClass({
     return (
       <View style={styles.fullscreenContainer}>
         {this.props.discoveryPanel}
-        {progressBar}
-        {controlBar}
```

Comment:
Those lines add the control bar, I think it's ok to remove them. But let's chat about what is the expected behavior.